config/samples/deploy_armada.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   ingress:
     ingressClass: "nginx"
   clusterIssuer: "k8s-cluster-issuer"
-  hostNames: [localhost]
+  hostNames: [localhost-server]
```

Comment:
Makes sense but these hostnames need to be unique. These are meaningless right now but we should make sure each service has a unique ip address.